torchaudio/functional.py

```diff
@@ -946,7 +957,7 @@ def gain(waveform, gain_db=1.0):
     r"""Apply amplification or attenuation to the whole waveform.
 
     Args:
-       waveform (torch.Tensor): Tensor of audio of dimension (channel, time).
+       waveform (torch.Tensor): Tensor of audio of dimension (..., time).
```

Comment:
Maybe ```([...], channel, time)``` to indicate that we still follow that convention of channel x time?

Reply:
The `...` is described in the [readme](https://github.com/pytorch/audio/blob/master/README.md), so I would not use `[...]` as we discussed in #337.

We can add `channel` everywhere to emphasize the convention, but that seems repetitive to me, so I wouldn't go for it. Thoughts/

Reply:
As long as it's documented and standardized so we can apply it consistently this is fine by me